easybuild/easyconfigs/j/Java/Java-1.8-8u265-b01-aarch64.eb

```diff
@@ -0,0 +1,15 @@
+name = 'Java'
+version = '1.8'
+versionsuffix = '-8u265-b01-aarch64'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/AdoptOpenJDK/openjdk8-binaries/releases/download/jdk8u265-b01/']
+sources = ['OpenJDK8U-jdk_aarch64_linux_hotspot_8u265b01.tar.gz']
```

Comment:
@terjekv There should be a relation with `version` here (which should be more than just `1.8`, as @Micket already suggested); same in `source_urls`.

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/11545/commits/44f8af7ba2dcd9a87366df32a2563a32e5fd80f9. Maybe?  :)